packages/checkout/blocks-registry/README.md

```diff
@@ -1,6 +1,6 @@
 # Checkout - Blocks Registry <!-- omit in toc -->
 
-This directory contains the Checkout Blocks Registry. This provides functions to **register new Inner Blocks** that can be inserted automatically, or optionally, within the cart and checkout blocks in certain areas.
+This directory contains the Checkout Blocks Registry. This provides functions to **register new Inner Blocks** that can be inserted automatically, or optionally, within the Cart and Checkout blocks in certain areas.
```

Comment:
I'm not sure about the functionality, but just checking that Mini Cart was not supposed to be here.

Reply:
Right, it looks like [there are some Mini Cart inner blocks areas](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/c6c24aa15984aa7d3e28632f6f779724ca60aaf6/packages/checkout/blocks-registry/types.ts#L22-L25). Updated in 21754b43ad5a4be9f42063fddcb7c17150a0c613.